tests/hackathon/checker.lua

```diff
@@ -0,0 +1,8 @@
+-- Copyright (c) Microsoft Corporation. All rights reserved.
+-- Licensed under the Apache 2.0 License.
+
+if (tx_type == 2 or tx_type == 3) and tonumber(amt) > 200000 then 
```

Comment:
2 minor questions:

- Why are the tx_types converted to numbers, it looks from the TransactionType enum like they were strings in the original csv files? The strings (or at least names for these in the environment?) would make the checker more readable.
- Is this the only regulator algorithm we have?

Reply:
- They are strings originally. It's very easy to change it back to strings and remove the enum. @jumaffre do you remember why it was converted to a number initially/do we need it somewhere?
- Yes this is the only regulator algorithm we have

Reply:
Regarding the algorithm, there's another one directly embedded in the `loader.py` for the second regulator (around line 68).